### ISSUE TITLE
p5.Amplitude example throws "ReferenceError: mouseClicked is not defined"

### ISSUE THREAD

**Issue by toolness** [processing/p5.js-website#188]
The [p5.Amplitude docs](http://p5js.org/reference/#/p5.Amplitude) throw this error and it seems to come from some code in `render.js` that is doing some [kind of funky stuff](https://github.com/processing/p5.js-website/blob/ff18b48612df460d63cc84fb11ac54e657c4db0f/js/render.js#L166-L173).

Unfortunately, it also looks like that exception is derailing the page code so it renders *really* weirdly:

![2015-11-08_13-57-08](https://cloud.githubusercontent.com/assets/124687/11021981/b9f6998c-8620-11e5-9dda-c4b750815ebf.png)

Removing the `cnv.mouseClicked` code from the example snippet makes everything render ok, at least.

Let me know if you want me to look into this further.

**Comment by lmccart** [comment]
sure have a look if you have time! some context, that hack was added in because the second example on [this page](http://p5js.org/reference/#p5/loadFont) was breaking because `drawText` had `draw` in the name and it was messing it up. I think better regex search for function names could fix this.

**Comment by therewasaguy** [comment]
cnv.mouseClicked used to work. I'm guessing it stopped working around the time render2D was introduced, maybe because the canvas is no longer a p5.Element? @lmccart should I look into bringing p5.Element events back to the canvas? It would be really useful for these examples where I want to toggle sound by clicking on the canvas. But I don't want the part of the code that makes the canvas clickable to be distracting.

**Comment by toolness** [comment]
Oh, @therewasaguy, that sketch works fine and `cnv.mouseClicked` still works--it's just the code in `render.js` is mistaking it for being a top-level function called `mouseClicked`, which is then causing an exception to be thrown.

TL;DR it has everything to do with the unusual way those example sketches are evaluated on the reference page, and nothing to do with the correctness of the example sketch itself :grin: My fix in #190 makes the example work again.

**Comment by lmccart** [comment]
the canvas is still a p5.Element I believe.. (p5.RendererXD < p5.Renderer < p5.Element)

**Comment by therewasaguy** [comment]
Ah, sorry for my confusion, thanks for explaining and the fix!
